src/Wpf.Ui/Controls/ListView/ListView.cs

```diff
@@ -0,0 +1,60 @@
+namespace Wpf.Ui.Controls;
+
+public class ListView : System.Windows.Controls.ListView
```

Comment:
New control should be added to https://github.com/lepoco/wpfui/blob/baaf749864df48be57c18821c9dcd47a2b5faa73/src/Wpf.Ui/VisualStudioToolsManifest.xml